src/render_tasks/d3d12_bloom_eighth_res.hpp

```diff
@@ -0,0 +1,169 @@
+#pragma once
```

Comment:
bloom_half , quarter, eighth and sixteenth tasks are all duplicate tasks with slight modifications to resolution and blurring factors.

Reply:
Consider what we talked about in the lab with Viktor about this approach. Please create an issue in github so we don't forget to optimize this in the future.